app/client/src/components/GenerateMicroreactURL.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <div>
+    <div>
+      <button
+        v-if='!microreactToken'
+        @click='showModal()'
+        class='btn btn-block btn-standard btn-download'
+      >
+        Generate Microreact URL
+      </button>
+
+      <button
+        v-if='microreactToken && !results.perCluster[cluster]?.microreactURL'
```

Comment:
You could simplify this by adding the code in computed method, so you will only need to call the computed method here instead of writing the whole line. It encourages readability.